src/deck-options.md

```diff
@@ -374,6 +374,7 @@ If you want to spend less time on Anki on some days of the week, such as Sundays
 After the optimal interval is calculated, it will be adjusted by a small amount to change the due date.
 Note that setting all days to "Reduced" or "Minimum" will result in the same workload as setting all days to "Normal".
 This feature works with both FSRS and the legacy SM-2 algorithm.
```

Comment:
Do we even need to document this?

Reply:
Yes, this is important. Some people might assume that it's FSRS-specific.

Reply:
I personally don't see the need, as the options screen automatically hides/shows options as FSRS is toggled. But I don't feel strongly about it.